File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex22.c
#include <stdio.h>

main()
{
    //vars
    int lados, diagonais;

    printf("\t\t\tNumero de diagonais de um poligono convexo\n\n");

    //Entrada
    printf("Insira o numero de lados do poligono: ");
    scanf("%d",&lados);

    //Processamento
    diagonais = (lados * (lados - 3)) / 2;

    //Saída
    printf("Numero de diagonais: %i", diagonais);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex22.java
//Importando Scanner
import java.util.Scanner;

public class Ex22 {

	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	int lados, diagonais;
	
	System.out.println("Número de diagonais de um polígono convexo\n\n");
	
	//Entradas	
	System.out.print("Insira o número de lados do polígono: ");
	lados = input.nextInt();
	
	//Processamento
	diagonais = lados * ( lados - 3 ) / 2;
	
	//Saída
	System.out.println("Número de diagonias: " + diagonais);
	
		
	}
}
